ultra/ultra/scenarios/README.md

```diff
@@ -54,14 +60,19 @@
     ``` 
 3. Generate scenarios for training:
     ```sh
-    $ python ultra/scenarios/interface.py generate --task 1 --level easy
+    # Remove existing easy scenarios.
+    $ rm -rf ultra/scenarios/task1/t*_easy_*
+
+    # Generate the analysis scenarios from the analysis maps in the analysis pool.
+    $ python ultra/scenarios/interface.py generate --task 1 --level easy --pool-dir ultra/scenarios/pool/analysis_pool/
     ```
     Available parameters:
     - `--task`: Selects the task to generate (based on the task's config.yaml).
     - `--level`: Selects the level of the task from the task's config.yaml.
     - `--stopwatcher`: Include a stopwatcher (a vehicle that records the number of steps taken to do the left turn) in the scenarios.
     - `--save-dir`: The directory for saving the completed scenarios (completed scenarios will be put in the task directory if not specified).
     - `--root-dir`: The directory containing the task to be created (default is ultra/scenarios).
+    - `--pool-dir`: The directory containing the maps for the scenarios (default is pool/experiment_pool).
```

Comment:
This would be changed too if the default value for the `--pool-dir` argument is changed.

Reply:
a1766c9